geometry/Overview.bs

```diff
@@ -714,11 +714,13 @@ interface DOMMatrixReadOnly {
                                     optional unrestricted double ty = 0,
                                     optional unrestricted double tz = 0);
     [NewObject] DOMMatrix scale(optional unrestricted double scaleX = 1,
-                                optional unrestricted double scaleY,
+                                optional unrestricted double scaleY = 1,
```

Comment:
Can you make this a separate issue/pr? I don't recall if this was intentional but that should be considered properly.